### ISSUE TITLE
Demo broken by private Widevine extension

### ISSUE THREAD

**Issue by innaignat** [shaka-project/shaka-player#6613]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**

**If the problem is related to FairPlay, have you read the tutorial?**

**What version of Shaka Player are you using?**

Shaka v4.8.4(unkompiled)

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `main`?**

**Are you using the demo app or your own custom app?**
Demo

**If custom app, can you reproduce the issue using our demo app?**

**What browser and OS are you using?**
CDM 4.10.2710.0, all Chrome browser versions, all platforms.

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**

**What are the manifest and license server URIs?**

asset=https://storage.googleapis.com/wvtemp/hmchen/1080p24fpsVP9_hlg_hdr_cenc.mpd;
license=https://proxy.uat.widevine.com/proxy

**What configuration are you using?  What is the output of `player.getConfiguration()`?**

**What did you do?**

Use Play Link:
https://shaka-player-demo.appspot.com/demo/#audiolang=en-US;textlang=en-US;clockSyncUri=;uilang=en-US;asset=https://storage.googleapis.com/wvtemp/hmchen/1080p24fpsVP9_hlg_hdr_cenc.mpd;license=https://proxy.uat.widevine.com/proxy;panel=CUSTOM%20CONTENT;build=uncompiled

**What did you expect to happen?**
Content plays acceptably

**What actually happened?**

Playback doesn't start.
The error message:
```
StringUtils.htmlUnescape is not a function
```

**Are you planning send a PR to fix it?**

**Comment by innaignat** [comment]
I have two more links to test Videostack Merge - transiting from encrypted to clear and back.

https://shaka-player-demo.appspot.com/demo/#audiolang=en-AU;asset=https://media.axprod.net/TestVectors/v7-MultiDRM-MultiKey-MultiPeriod/Manifest.mpd

https://shaka-player-demo.appspot.com/demo/#audiolang=en-ET;asset=https://media.axprod.net/TestVectors/v7-MultiDRM-MultiKey-MultiPeriod/Manifest.mpd

 I got the same error for those tests.

**Comment by joeyparrish** [comment]
The Axinom v6 and v7 test vectors are likely no longer being hosted.  Please send a PR to update them!  See https://github.com/Axinom/public-test-vectors and https://github.com/shaka-project/shaka-player/blob/main/demo/common/assets.js

**Comment by theodab** [comment]
I'm not reproducing this on my computer, and this doesn't look like the kind of issue that would be platform-specific. The PR that moved the `htmlUnescape` method from `VttTextParser` to `StringUtils` also changed all references to the method at the same time...

My first thought is it might be an individual source file being cached on your system, or something like that? It might be possible with the uncompiled demo, which you linked in your report.
Does the problem still happen if you run the compiled demo?

**Comment by innaignat** [comment]
I replaced `build=uncompiled` with `build=compiled` in the test link. Now it works as expected. I just used old test links for tests using Shaka Player. Now I will change them all.

**Comment by joeyparrish** [comment]
I realized that the OP is in our office, so I sat down with her to investigate in person.  The issue was caused by a private Chrome extension used by Widevine for debugging.  It appears to break the demo in uncompiled mode, though it's not clear how or why.

Switching to compiled mode is an effective workaround until Widevine fixes that private extension.

@xhwang-chromium JFYI.

In any case, disabling this Widevine debugging extension makes the player and demo work correctly, so there is no Shaka bug or Shaka demo bug here.
